src/app/services/breathecode.js

```diff
@@ -1182,6 +1182,19 @@ class BreatheCodeClient {
     };
   }
 
+  payments() {
+    return {
+      addAcademyPlanSlugSubscription: (planSlug, payload) =>
+        axios.bcPost(
+          "Academy Plan Subscription",
+          `${this.host}/payments/academy/plan/${planSlug}/subscription`,
+          payload
+        ),
+      getPlanByCohort: (cohortSlug) =>
+        axios.get(`${config.REACT_APP_API_HOST}/v1/payments/plan?cohort=${cohortSlug}`),   
```

Comment:
This should be getPlan and it should receive the queries for additional filters as another parameter